### DISCUSSION TITLE
Plugin stopped working in Version 5.6.0 Woocommerce

### DISCUSSION THREAD

**Discussion by arrenato** [lukecav/performance-improvements-for-woocommerce#11] [General]
Hello, with the new Woocommerce update some plugin features stop working eg disabling top bar woocommerce. Thanks

![top bar](https://user-images.githubusercontent.com/5803751/132574119-494ffc3d-d666-436d-90cb-581b59e45cad.png)

**Comment by lukecav** [comment]
Let me take a look.
